src/reconcile/rec_write.c

```diff
@@ -1290,8 +1309,8 @@ __rec_txn_read(WT_SESSION_IMPL *session, WT_RECONCILE *r,
 		}
```

Comment:
Also, unless we were to leave a reserved update with a transaction ID after that transaction commits, or checkpoint were to start calling `WT_CURSOR::reserve`, it should never see a reserved update visible.